internal/clocktest/clocktest.go

```diff
@@ -80,7 +80,15 @@ func (f fakeClock) NewTicker(d time.Duration) internal.Ticker {
 // clockwork.Clock.NewTimer as a clock.Timer. See package comment for more
 // information on why this is necessary.
 func (f fakeClock) NewTimer(d time.Duration) internal.Timer {
-	return f.clockworkFakeClock.NewTimer(d)
+	t := f.clockworkFakeClock.NewTimer(d)
```

Comment:
Name it `timer` to fix this lint issue.

BTW, you can run the linter locally before pushing via `make lint`.

Reply:
Got it, now it should be fixed